metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy

```diff
@@ -123,6 +123,16 @@ class MetacatSmokeSpec extends Specification {
             if (metadata != null) {
                 newTable.getMetadata() == null? newTable.setMetadata(metadata): newTable.getMetadata().putAll(metadata)
             }
+
+            if (tableName == 's3-mysql-db') {
```

Comment:
Would it make sense to just set the owner for all tables created

Reply:
It is mandated only for the S3 connector, but let me try and see what breaks.